### ISSUE TITLE
Bug located in modeler.points

### ISSUE THREAD

**Issue by maxcapodi78** [pyansys/pyaedt#1941]
### Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### Description of the bug

Points are not correctly updated.

### Steps To Reproduce

on existing project the points where not correctly updated.

### Which Operating System are you using?

Windows

### Which Python version are you using?

3.10

### Installed packages

aiohttp==3.8.1
aiosignal==1.2.0
alabaster==0.7.12
altgraph==0.17.2
ansys-sphinx-theme==0.4.2
anyio==3.6.1
appdirs==1.4.4
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
asttokens==2.0.5
async-timeout==4.0.2
atomicwrites==1.4.0
attrs==21.4.0
auto-py-to-exe==2.22.0
Babel==2.10.3
backcall==0.2.0
bcrypt==3.2.2
beautifulsoup4==4.11.1
bleach==5.0.0
bottle==0.12.23
bottle-websocket==0.2.9
certifi==2022.6.15
cffi==1.15.0
charset-normalizer==2.0.12
clang==14.0
clr-loader==0.2.3
codecov==2.1.12
colorama==0.4.5
commonmark==0.9.1
contourpy==1.0.5
coverage==6.4.1
cryptography==37.0.4
cycler==0.11.0
debugpy==1.6.0
decorator==5.1.1
defusedxml==0.7.1
distro==1.8.0
docutils==0.17.1
dotnetcore2==3.1.23
Eel==0.14.0
entrypoints==0.4
et-xmlfile==1.1.0
execnet==1.9.0
executing==0.8.3
fastjsonschema==2.15.3
fonttools==4.33.3
frozenlist==1.3.0
future==0.18.2
gevent==21.12.0
gevent-websocket==0.10.1
greenlet==1.1.3
idna==3.3
imageio==2.19.3
imageio-ffmpeg==0.4.7
imagesize==1.4.1
iniconfig==1.1.1
ipykernel==6.15.0
ipython==8.4.0
ipython-genutils==0.2.0
ipywidgets==7.7.1
jedi==0.18.1
Jinja2==3.1.2
joblib==1.2.0
json5==0.9.8
jsonschema==4.6.0
jupyter-client==7.3.4
jupyter-core==4.10.0
jupyter-server==1.17.1
jupyterlab==3.4.3
jupyterlab-pygments==0.2.2
jupyterlab-server==2.14.0
jupyterlab-widgets==1.1.1
kiwisolver==1.4.3
livereload==2.6.3
MarkupSafe==2.1.1
matplotlib==3.6.1
matplotlib-inline==0.1.3
mistune==0.8.4
multidict==6.0.2
mysql==0.0.3
mysqlclient==2.1.1
nbclassic==0.3.7
nbclient==0.6.4
nbconvert==6.5.0
nbformat==5.4.0
nbsphinx==0.8.9
nest-asyncio==1.5.5
notebook==6.4.12
notebook-shim==0.1.0
numpy==1.23.3
numpydoc==1.4.0
openpyxl==3.0.10
packaging==21.3
pandas==1.5.0
pandocfilters==1.5.0
paramiko==2.11.0
parso==0.8.3
pefile==2022.5.30
pickleshare==0.7.5
Pillow==9.1.1
pluggy==1.0.0
plumbum==1.7.2
prometheus-client==0.14.1
prompt-toolkit==3.0.29
psutil==5.9.1
pure-eval==0.2.2
py==1.11.0
pyansys-sphinx-theme==0.3.1
pycparser==2.21
pydata-sphinx-theme==0.9.0
Pygments==2.12.0
pyinstaller==5.3
pyinstaller-hooks-contrib==2022.8
PyNaCl==1.5.0
pypandoc==1.8.1
pyparsing==3.0.9
pyrsistent==0.18.1
pytest==7.1.3
pytest-cov==3.0.0
pytest-forked==1.4.0
pytest-html==3.1.1
pytest-metadata==2.0.2
pytest-sphinx==0.4.0
pytest-xdist==2.5.0
python-dateutil==2.8.2
pythonnet==3.0.0.post1
pytz==2022.1
pyvista==0.36.1
pywin32==303
pywin32-ctypes==0.2.0
pywinpty==2.0.5
pyzmq==23.2.0
recommonmark==0.7.1
requests==2.28.0
rpyc==5.0.1
scikit-learn==1.1.2
scipy==1.8.1
scooby==0.5.12
Send2Trash==1.8.0
six==1.16.0
sklearn==0.0
sniffio==1.2.0
snowballstemmer==2.2.0
soupsieve==2.3.2.post1
Sphinx==5.0.2
sphinx-autobuild==2021.3.14
sphinx-autodoc-typehints==1.18.3
sphinx-copybutton==0.5.0
sphinx-gallery==0.10.1
sphinx-notfound-page==0.8
sphinxcontrib-applehelp==1.0.2
sphinxcontrib-devhelp==1.0.2
sphinxcontrib-htmlhelp==2.0.0
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.3
sphinxcontrib-serializinghtml==1.1.5
sphinxcontrib-websupport==1.2.4
stack-data==0.3.0
terminado==0.15.0
threadpoolctl==3.1.0
tinycss2==1.1.1
tomli==2.0.1
tornado==6.1
traitlets==5.3.0
urllib3==1.26.9
vtk==9.2.0rc1
wcwidth==0.2.5
webencodings==0.5.1
websocket-client==1.3.3
whichcraft==0.6.1
widgetsnbextension==3.6.1
wslink==1.6.5
yarl==1.7.2
zope.event==4.5.0
zope.interface==5.4.0
